### PR TITLE
Fix duplicated id for review tables

### PR DESCRIPTION
**Pull request by SD1982**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | The tables "waiting for approval" , "reported reviews" and "approved reviews" all have the same id this PR fix this
| Type?         | bug fix 
| BC breaks?    | no 
| Deprecations? | no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/24528
| How to test?  | check the product comments module configuration page, everything should work as before, and also check the html in browser, the id should be different for each table

### CODE REVIEW

**Review by matthieu-rolland — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by hibatallahAouadni** [comment]
Why there's no label **`Bug Fix`**? :confused:

**Review by Progi1984 — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @SD1982 !

The ID are now OK, but I can no longer delete reviews in the different lists (reviews waiting for approval, reported reviews, approved reviews).

Could you check that ? Thanks !

**Review by atomiix — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @SD1982 

Now it's OK, the ID are OK and deletion and display of item is OK : QA :heavy_check_mark: 

Thanks !

**Comment by matks** [comment]
Thank you @SD1982

**Pull request merged**
